openhands/server/shared.py

```diff
@@ -19,6 +19,12 @@
 server_config = load_server_config()
 file_store = get_file_store(config.file_store, config.file_store_path)
 
+from openhands.storage.docker_snapshots import sudo_command
+if config.sandbox.docker_snapshots:
+    sudo_command(["enable"])
+else:
+    sudo_command(["disable"])
+
```

Comment:
this definitely shouldn't go in `shared.py`--this doesn't really have anything to do with the server. Maybe it should go in docker_runtime.py?